src/NTestController/NTestController.cs

```diff
@@ -74,6 +97,33 @@ public static void Main(string[] args)
 
         #region Private functions
 
+        private static void ExecuteTests(IExecutorPlugin executorPlugin, List<IPlatform> platforms)
+        {
+            ThrowIf.ArgumentNull(executorPlugin, nameof(executorPlugin));
+            ThrowIf.ArgumentNull(platforms, nameof(platforms));
+
+            // Hack: For now just use a single Platform.  TODO: Use all Platforms later.
+            var firstPlatform = platforms[0];
+            var plugins = new List<IExecutorPlugin> { executorPlugin };
+
+            executorPlugin.Computer = firstPlatform.Computers[0];
+
+            // We already have one plugin, so add 1 less than Computers.Count.
+            for (int i = 1; i < firstPlatform.Computers.Count; ++i)
+            {
+                var newPlugin = executorPlugin.ClonePlugin();

```

Comment:
Maybe add a comment explaining that we're cloning the Executor plugin here because we need each thread to execute against a different Computer.